tests/rptest/services/openmessaging_benchmark_configs.py

```diff
@@ -0,0 +1,170 @@
+    SIMPLE_DRIVER = {
+        "name": "simple-driver",
+        "replication_factor": 3,
+        "request_timeout": 120000,
+        "enable_idempotence": "false",
+        "acks": "1",
+        "linger_ms": 1,
+        "max_in_flight": 1,
+        "batch_size": 131072,
+        "auto_offset_earliest": "earliest",
+        "auto_commit": "false",
+        "max_partition_fetch": 131072
+    }
+
+    ACK_ALL_GROUP_LINGER_1MS = {
+        "name": "ack-all-group-linger-1-ms-no-idem",
+        "replication_factor": 3,
+        "request_timeout": 120000,
+        "enable_idempotence": "false",
+        "acks": "all",
+        "linger_ms": 1,
+        "max_in_flight": 1,
+        "batch_size": 131072,
+        "auto_offset_earliest": "earliest",
+        "auto_commit": "false",
+        "max_partition_fetch": 131072
+    }
+
+    ACK_ALL_GROUP_LINGER_1MS_WITH_IDEMPOTENCE = {
+        "name": "ack-all-group-linger-1-ms-enable-idem",
+        "replication_factor": 3,
+        "request_timeout": 120000,
+        "enable_idempotence": "true",
+        "acks": "all",
+        "linger_ms": 1,
+        "max_in_flight": 1,
+        "batch_size": 131072,
+        "auto_offset_earliest": "earliest",
+        "auto_commit": "false",
+        "max_partition_fetch": 131072
+    }
+
+    ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT = {
+        "name": "ack-all-group-linger-1-ms-enable-idem",
+        "replication_factor": 3,
+        "request_timeout": 120000,
+        "enable_idempotence": "true",
+        "acks": "all",
+        "linger_ms": 1,
+        "max_in_flight": 5,
+        "batch_size": 131072,
+        "auto_offset_earliest": "earliest",
+        "auto_commit": "false",
+        "max_partition_fetch": 131072
+    }
+
+    ACK_ALL_GROUP_LINGER_10MS = {
+        "name": "ack-all-group-linger-10-ms-no-idem",
+        "replication_factor": 3,
+        "request_timeout": 120000,
+        "enable_idempotence": "false",
+        "acks": "all",
+        "linger_ms": 10,
+        "max_in_flight": 1,
+        "batch_size": 131072,
+        "auto_offset_earliest": "earliest",
+        "auto_commit": "false",
+        "max_partition_fetch": 131072
+    }
+
+    ACK_ALL_GROUP_LINGER_10MS_WITH_IDEMPOTENCE = {
+        "name": "ack-all-group-linger-10-ms-enable-idem",
+        "replication_factor": 3,
+        "request_timeout": 120000,
+        "enable_idempotence": "true",
+        "acks": "all",
+        "linger_ms": 1,
+        "max_in_flight": 1,
+        "batch_size": 131072,
+        "auto_offset_earliest": "earliest",
+        "auto_commit": "false",
+        "max_partition_fetch": 131072
+    }
+
+    # ------ Driver configurations end--------
+
+    # ------- Workload configurations --------
+
+    TOPIC1_PART100_1KB_4PROD_1250K_RATE = {
+        "name": "100-partitions-1000K-rate-4-producer",
+        "topics": 1,
+        "partitions_per_topic": 100,
+        "subscriptions_per_topic": 1,
+        "consumer_per_subscription": 16,
```

Comment:
How it works with only 2 worker instances?
I thought that every consumer needs separate worker